style.css

```diff
@@ -0,0 +1,10 @@
+body {
+  margin-left: 10px;
+  padding: 10px;
+}
+
+@media screen and (max-width: 700px) {
+  body {
+    background-color: rgb(116, 143, 139);
+  }
+}
```

Comment:
It would be nice to be polite always when conducting reviews and to also give reasons why authors should make changes if it is required

Reply:
> Now let's remove the bullets and the margins and padding
> And you need to use Grid Elements properly.
- [ ] Hi thanks for implementing the previous reviewer's suggestion 👍. I still think this could be phrased a lot kinder than it is right now. Kindly try not to phrase your sentences as orders; at the end of the day, they are still suggestions. we don't want it to come across as being commanding.

Reply:
![Inked2021-11-30_02-23-42_LI](https://user-images.githubusercontent.com/63915024/143945701-d03a7304-c85d-4df6-903e-9ef2cec60e7c.jpg)
**LEFT Picture** is your work.                                                                                                                                          **RIGHT Picture** is required Template.



Please consider removing the bullets and the margins and padding from the list. Please You should be able to do so by using "list-style-type:" property.
Please also consider to use
 ***Grid Elements*** properly.

I believe you can do it 💪! Thanks in advance for the work.😊